### ISSUE TITLE
REQUEST: Create github.com/kubernetes-csi/csi-controller REPO

### ISSUE THREAD

**Issue by lpabon** [kubernetes/org#1442]
### New Repo, Staging Repo, or migrate existing
Migrate github.com/lpabon/csi-controller

### Requested name for new repository
csi-controller

### Which Organization should it reside
github.com/kubernetes-csi

### If not a staging repo, who should have admin access
@msau42 , @lpabon, @jsafrane 

### If not a staging repo, who should have write access
None

### If not a staging repo, who should be listed as approvers in OWNERS
@msau42 , @lpabon , @jsafrane , @xing-yang , @saad-ali , @mlmhl , @gnufied, @pohly 

### If not a staging repo, who should be listed in SECURITY_CONTACTS
@msau42 , @lpabon , @jsafrane , @xing-yang , @saad-ali , @mlmhl , @gnufied 

### What should the repo description be
Single intelligent CSI controller for Kubernetes

### What SIG and subproject does this fall under in sigs.yaml
sig-storage

### Approvals
This was discussed at Kubecon with @jsafrane and @msau42 

### Additional context for request
None

**Comment by msau42** [comment]
@lpabon it would be good to reference the discussion on the sig-storage mailing list too.

For description, I would probably reword it to something like:
```
Single CSI sidecar that combines all the individual sidecars into one
```

**Comment by jsafrane** [comment]
/lgtm
(or +1, not sure about the process)

**Comment by mrbobbytables** [comment]
> (or +1, not sure about the process)

Usually there's some discussion on the sig ML, and general sign-off from sig-leads (@saad-ali). We need their ack that it is something the sig wants to maintain before moving forward with repo creation/migration.

**Comment by lpabon** [comment]
Gotcha, I will add to the mailing list.

**Comment by lpabon** [comment]
Gotcha, I will add to the mailing list.

**Comment by lpabon** [comment]
Closing issue. Please see mailing list for more information: https://groups.google.com/d/msg/kubernetes-sig-storage/BMLIZcGT2FU/WmNuG8CbAQAJ
